js/app/modules/layout/articleStack.js

```diff
@@ -81,6 +81,10 @@ const ArticleStack = new Module.Class({
         },
     },
 
+    References: {
+        'selection': {},  // type: Selection
```

Comment:
On further consideration, I think this reference should be made optional, since none of the other presets use it (yet).